### PR TITLE
Update tailsitter info for BiCopter

### PR DESCRIPTION
**Pull request by Hwurzburg**
Pete, another question...does BiCopter use the Q_TILT_YAW_ANGLE param? seems like it would have to ....but I assume the other transition parameters are exactly like the tailsitter transition stuff and not the tilt rotor transition stuff?

### CODE REVIEW

**Comment by IamPete1** [comment]
Servo min is the forward flight position, trim is up, and max is back. So there should be 90 deg of moment between min (forward) and trim (up). Q_TILT_YAW_ANGLE should be set to the angle between the up (trim) and back (max).

**Comment by Hwurzburg** [comment]
I dont think trim is involved in normal tilt rotor setup.....set the tilt yaw angle, then adjust max for up (its really not that PWM..you are adjusting up+yaw angle with max, but QSTAB drives the up) and min for forward (manual)....dont think trim value is involved or has any effect....so this is different

**Comment by kd0aij** [comment]
I'm confused; are we talking about tiltrotors or copters here? 
The only SITL vehicle I have is a copter; is there an RF8 model configured as a bicopter for quadplane?

**Review by kd0aij — COMMENTED** [review]
the yaw torque comment looks good to me
But the bicopter/tailsitter thing seems confusing, especially since the V22 is not a tailsitter.

**Comment by Hwurzburg** [comment]
the yaw torque comment is wrong you just told me....stable does not have it (no yaw torque...it has it)....or maybe, it needs a stable number as to when it was added to stable

**Comment by kd0aij** [comment]
What's confusing is that a bicopter tailsitter is not a tailsitter, but it uses frame class 10.  It might be better to define a new frame class for the V22, just to avoid confusion.

**Comment by Hwurzburg** [comment]
does it use Quadplane transitions or Tailsitter transitions and params?if it uses thr-lft/rgt I expect it uses tailsitter transitions...very confusing....hence the need for a clear wiki

**Comment by IamPete1** [comment]
It does standard quad-plane transmissions. I think the confusion is simply that the 'bicopter' frame type is called tailsitter because that is what it was original added for. 

So you can have a 'bi-copter tailsitter' or a 'bi-copter tilt-rottor' or in ArduCopter  just a 'bi-copter'. 

We probably should look at separating the tailsitter and tilt rotor types from frame types, now you can have more combinations of each.

The tilt and tailsitter mask are the current tirggers, but the bicopter frame type does not use the same masks so needs a special case for each.

**Comment by Hwurzburg** [comment]
After talking with Pete, and finally understanding how BiCopter is setup, I have added a BiCopter section and modified previous entries....

**Review by kd0aij — APPROVED** [review]
the bicopter tiltrotor stuff sounds good to me

File: plane/source/docs/guide-tilt-rotor.rst [hunk 1]
```diff
@@ -66,9 +65,11 @@ hovering. Currently supported tilt-rotor frame classes are:
    <tr><td>Tricopter</td><td>7</td></tr>
    </table>
 
+.. note: the BiCopter tilt-rotor (either motors tilting ,or entire wing with fixed motors tilting) QuadPlane is a special case. It requires that :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` be set to 10 which is reserved for Tailsitter configurations only, normally. See :ref:`BiCopter <bicopter>`  below.
```
Comment (IamPete1) [inline plane/source/docs/guide-tilt-rotor.rst:68]:
```suggestion
.. note: the BiCopter tilt-rotor QuadPlane is a special case. It requires that :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` be set to 10 which is reserved for Tailsitter configurations only, normally. See :ref:`BiCopter <bicopter>`  below.
```

Bi-copter needs its two motors to be able to tilt interdependently.

Comment (Hwurzburg) [reply plane/source/docs/guide-tilt-rotor.rst]: fixed

**Review by IamPete1 — APPROVED** [review]

**Comment by Hwurzburg** [comment]
thanks! merging

**Pull request merged**